ee/packages/license/src/events/emitter.ts

```diff
@@ -2,13 +2,17 @@ import type { BehaviorWithContext, LicenseModule } from '@rocket.chat/core-typin
 
 import type { LicenseManager } from '../license';
 import { logger } from '../logger';
+import { isInternalModuleName } from '../modules';
 
 export function moduleValidated(this: LicenseManager, module: LicenseModule) {
 	try {
-		this.emit('module', { module, valid: true });
+		const external = !isInternalModuleName(module);
```

Comment:
Ideally this place would have access to the full module definition to pass on to the event instead of having to determine by itself if the module is external or not, but that might be a PITA to implement now (specially for the module removed event) so I guess we can leave it as is for now and just refactor it if and only if we ever add any other attribute to the definition.